docs/connector-development/README.md

```diff
@@ -131,7 +131,8 @@ Once you've finished iterating on the changes to a connector as specified in its
    
    * Then run the command `./gradlew :airbyte-config:init:processResources` to generate the seed spec yaml files, and commit the changes to the PR. See [this readme](https://github.com/airbytehq/airbyte/tree/a534bb2a8f29b20e3cc7c52fef1bc3c34783695d/airbyte-config/specs) for more information.
    
-5. The new version of the connector is now available for everyone who uses it. Thank you!
+5. OPTIONAL: If the `README.md` file of the connector contains a `Changelog` section, add the new version and relevant release information to the table in the section.
```

Comment:
Should we keep it as `OPTIONAL`? It sounds that it should be mandatory. If there is no changelog we should add one IMO.

Reply:
@benmoriceau It's a bit odd -- not all connectors have a `Changelog`.  I suppose that we could just remove `OPTIONAL`, as the `If` covers the conditional case.